src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -9207,10 +9237,20 @@ public static void Main()
         <member name=""M:C.Main"">
             <summary>Main method</summary>
         </member>
+        <member name=""F:C.field"">
```

Comment:
Such fields are stripped from the ref assembly, but not from the doc output. I'll take a stab tomorrow to strip it out.